app/helpers/court_dates_helper.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Helper methods for court_dates
+module CourtDatesHelper
+  def when_do_we_have_court_dates(casa_case)
+    date_query = casa_case.court_dates.includes(:hearing_type).ordered_ascending
+    last_court_date = date_query.last
+    first_court_date = date_query.first
+
+    if last_court_date.date < Date.current
```

Comment:
**suggestion:** it shouldn't make a difference (unless a user is unlucky enough for this script to be executed at the exact moment the code is running), but if you want to ensure we're always using the date date, it'd be better to store `Date.current` in a variable and use it instead.